OrbitGl/Track.cpp

```diff
@@ -200,7 +200,8 @@ void Track::SetY(float y) {
 }
 
 Color Track::GetBackgroundColor() const {
-  int32_t capture_process_id = GOrbitApp->GetCaptureData().process_id();
+  const CaptureData* capture_data = time_graph_->GetCaptureData();
+  int32_t capture_process_id = capture_data ? capture_data->process_id() : 0;
```

Comment:
Don't we have -1 as "invalid process"? Also see line 207.

Reply:
Yes, thanks!